src/navigation/BottomTabNavigator.tsx

```diff
@@ -0,0 +1,93 @@
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs'
+import React from 'react'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import { useTheme } from 'styled-components/native'
+
+import {
+  BookOutlineIconGrey,
+  BookOutlineIconWhite,
+  HeartOutlineIconGrey,
+  HeartOutlineIconWhite,
+  HomeOutlineIconGrey,
+  HomeOutlineIconWhite,
+  StarOutlineIconGrey,
+  StarOutlineIconWhite
+} from '../../assets/images'
+import labels from '../constants/labels.json'
+import HomeStackNavigator from './HomeStackNavigator'
+import { RoutesParams } from './NavigationTypes'
+
+const BottomTabNavigator = createBottomTabNavigator<RoutesParams>()
+
+// TODO LUN-132, LUN-207, LUN-308: Remove this and use actual components
+const MockComponent = () => <></>
```

Comment:
Only valid options should be displayed. Please hide the bottom bar as long as only home is available.